api/src/services/restaurants/restaurants.js

```diff
@@ -26,7 +36,18 @@ export const updateRestaurant = ({ id, input }) => {
   requireAuth()
```

Comment:
Can we destructure the fields we need from `input`?
```suggestion
  requireAuth()
  
  const { name, brandColor, address } = input
```

Reply:
Is this type of destructuring correct? Seems like a lot of code..

` const onSubmit = ({
    addressNumber,
    addressStreet,
    city,
    country,
    postalCode,
    state,
    restaurantName,
    userName,
  }) => {
    create({
      variables: {
        input: {
          userName,
          restaurantName,
          brandColor: colorHex,
          address: {
            addressNumber,
            addressStreet,
            city,
            country,
            postalCode,
            state,
          },
        },
      },
    })
  }`